src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -0,0 +1,184 @@
+package com.bugsnag.android;
+
+import java.sql.BatchUpdateException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Used as the entry point for native code to allow proguard to obfuscate other areas if needed
+ */
+class NativeInterface {
+
+    public static String getContext() {
+        return Bugsnag.getClient().getContext();
```

Comment:
Refactored code to try and allow this to work with client objects without the Bugsnag class